dapps/digital-twin-detail/src/components/DigitalTwinDetail.vue

```diff
@@ -0,0 +1,130 @@
+  the following URL: https://evan.network/license/
+*/
+
+<template>
+  <div class="evan theme-evan" :id="$route.params.address">
+    <evan-dapp-wrapper>
+      <template v-slot:content>
+        <evan-dapp-wrapper-level-2 ref="level2Wrapper">
+          <div class="sidenav">
+            <div class="sidenav-header">
+              <div class="icon-row">
+                <evan-button @click="close" :type="'icon-secondary'" icon="mdi mdi-close" />
+                <div class="flex-grow-1"></div>
+                <evan-button :type="'icon-secondary'" icon="mdi mdi-star-outline" />
+                <b-dropdown variant="link" toggle-class="text-decoration-none" no-caret>
+                  <template v-slot:button-content>
+                    <evan-button
+                      :type="'icon-secondary'"
+                      icon="mdi mdi-dots-vertical"
+                    />
+                  </template>
+                  <b-dropdown-item href="#">TODO Clone</b-dropdown-item>
+                  <b-dropdown-item href="#">TODO Export</b-dropdown-item>
+                  <b-dropdown-item href="#">TODO Remove</b-dropdown-item>
+                </b-dropdown>
+              </div>
+
+              <evan-profile-picture
+                class="twin-avatar"
+                type="device"
+                :src="'https://via.placeholder.com/96'"
+              />
+              <h4 class="twin-name text-center mt-2">TODO NAME</h4>
+              <h5 class="twin-owner text-center">TODO OWNER</h5>
+              <small
+                class="twin-desc text-center mt-3"
+              >TODO This is a brief description of the specific Digital Twin. It may also include application tips and recommendations for action..</small>
+            </div>
+
+            <!-- Not using nav-list because it doesnt support router-link properly
+            TODO: Refactor evan-nav-list to use router-links too-->
+            <div class="evan-nav-list">
+              <div class="nav-entries">
+                <template v-for="navItem in navItems">
+                  <router-link
+                    :id="navItem.id"
+                    :key="navItem.id"
+                    :to="navItem.to"
+                    :active-class="'active'"
+                  >
+                    <i class="mr-3" :class="navItem.icon"></i>
+                    {{ navItem.label | translate }}
+                  </router-link>
+                </template>
+              </div>
+            </div>
+          </div>
+        </evan-dapp-wrapper-level-2>
+        <transition name="fade" mode="out-in">
+          <router-view></router-view>
+        </transition>
+      </template>
+    </evan-dapp-wrapper>
+  </div>
+</template>
+
+<script lang="ts">
+import DigitalTwinDetailComponent from './DigitalTwinDetail';
+export default DigitalTwinDetailComponent;
+</script>
+
+<style lang="scss" scoped>
+@import '~@evan.network/ui/src/style/utils';
```

Comment:
no DigitalTwinDetail.scss  file 😢

Reply:
I think we agreed for files > 200 LOC? :p